app/repositories/group.js

```diff
@@ -0,0 +1,25 @@
+var mongoose = require( 'mongoose' );
+var Group = require( '../models/group.js' );
+var _ = require( 'lodash' );
+
+/**
+ * This method is used to create new group with the given params
+ *
+ * @param {Object} groupParams - You can take the schema of this object from Group model
+ * @param res - The respond of the HTTP request
+ */
+module.exports.createGroup = function( groupParams, res ) {
```

Comment:
Here the suffix create**Group** is  unnecessary, because it is infer by Group repository context.